docs/data/base/components/button/UnstyledButtonIntroduction/system/index.js

```diff
@@ -58,23 +58,21 @@ const Button = styled(BaseButton)(
   &:active {
     background-color: ${blue[700]};
     box-shadow: none;
+    transform: scale(0.99);
   }
 
   &:focus-visible {
     box-shadow: 0 0 0 4px ${theme.palette.mode === 'dark' ? blue[300] : blue[200]};
     outline: none;
   }
 
-  &:disabled {
+  &.Mui-disabled {
```

Comment:
Fix 3.

Reply:
Let's not use class names as strings but reference them from `buttonClasses`.

Reply:
OK